src/Symfony/Bundle/FrameworkBundle/Tests/Kernel/MicroKernelTraitTest.php

```diff
@@ -184,4 +185,34 @@ public function testDefaultKernel()
 
         $this->assertSame('OK', $response->getContent());
     }
+
+    public function testGetKernelParameters()
+    {
+        $kernel = $this->kernel = new ConcreteMicroKernel('test', false);
+
+        $parameters = $kernel->getKernelParameters();
+
+        $this->assertSame($kernel->getConfigDir(), $parameters['.kernel.config_dir']);
+        $this->assertSame(['test'], $parameters['.kernel.known_envs']);
+        $this->assertSame(['Symfony\Bundle\FrameworkBundle\FrameworkBundle' => ['all' => true]], $parameters['.kernel.all_bundles']);
+    }
+
+    public function testGetKernelParametersWithBundlesFile()
+    {
+        $kernel = $this->kernel = new ConcreteMicroKernel('test', false);
+
+        $configDir = $kernel->getConfigDir();
+        mkdir($configDir, 0o777, true);
+
+        $bundlesContent = "<?php\nreturn [\n    'Symfony\Bundle\FrameworkBundle\FrameworkBundle' => ['all' => true],\n    'TestBundle' => ['test' => true, 'dev' => true],\n];";
+        file_put_contents($configDir.'/bundles.php', $bundlesContent);
```

Comment:
Nit: PHPStorm does syntax highlighting and static analysis inside heredocs when the language name is used as delimiter. https://www.jetbrains.com/help/phpstorm/using-language-injections.html#inject-language-nowdoc-heredoc
```suggestion
        file_put_contents($configDir.'/bundles.php', <<<'PHP'
        <?php
        return [
            'Symfony\Bundle\FrameworkBundle\FrameworkBundle' => ['all' => true],
            'TestBundle' => ['test' => true, 'dev' => true],
        ];
        PHP);
```